### PR TITLE
Add an endpoint to place orders

### PR DESCRIPTION
**Pull request by jrbartola**
**Overview**

Addresses #29 using @dahifi 's solution. Adds a method to the API client allowing one to POST orders to the API.

### CODE REVIEW

**Comment by timkpaine** [comment]
@jrbartola looks good, can you flake8 it so it passes lint?

File: tdameritrade/client.py [hunk 1]
```diff
@@ -116,3 +116,8 @@ def movers(self, index, direction='up', change_type='percent'):
                             headers=self._headers(),
                             params={'direction': direction,
                                     'change_type': change_type}).json()
+
+    def orders(self, account_id, order):
+        return requests.post(ACCOUNTS + account_id + "/orders",
+                             headers=self._headers(),
+                             json=order).json()
```
Comment (arh) [inline tdameritrade/client.py]: You need to remove `.json()` in here. It only works if the request get an error code. If there is no error and request returned with code 201, there is nothing to convert it to by `json()`.

Comment (jrbartola) [reply tdameritrade/client.py]: Good catch! I’ll fix it in a bit

**Comment by dahifi** [comment]
The saved_orders endpoint is exactly the same function, may as well add that as well.

**Pull request merged**